modules/cluster-logging-collector-log-forward-es.adoc

```diff
@@ -64,7 +64,7 @@ spec:
 <5> Specify the `elasticsearch` type.
 <6> Specify the URL and port of the external Elasticsearch instance as a valid absolute URL. You can use the `http` (insecure) or `https` (secure HTTP) protocol. If the cluster-wide proxy using the CIDR annotation is enabled, the output must be a server name or FQDN, not an IP Address.
 <7> For a secure connection, you can specify an `https` or `http` URL that you authenticate by specifying a `secret`.
-<8> For an `https` prefix, specify the name of the secret required by the endpoint for TLS communication. The secret must exist in the `openshift-logging` project, and must have keys of *tls.crt*, *tls.key*, and *ca-bundle.crt* that point to the respective certificates that they represent. Otherwise, for `http` and `https` prefixes, you can specify a secret that contains a username and password. For more information, see the following "Example: Setting a secret that contains a username and password."
+<8> For an `https` prefix, specify the name of the secret required by the endpoint for TLS communication. The secret must exist in the `openshift-logging` project and must contain a `ca-bundle.crt` key that points to the certificate it represents. Otherwise, for `http` and `https` prefixes, you can specify a secret that contains a username and password. For more information, see the following "Example: Setting a secret that contains a username and password."
```

Comment:
I suggest removing these words "The secret must exist in the `openshift-logging` project" .  For mCFL, the secret  isn't in Openshift-logging namespace.

Reply:
Updated to say "In legacy implementations, the secret must exist in the `openshift-logging` project." since this would still apply for older versions.